Tests/Client/RestClientTest.php

```diff
@@ -285,11 +285,11 @@ public function testErrorCodesIgnoreInvalidResponse()
         $logger = new Logger("test", [$handler]);
         $client = new RestClient($logger, [], [], [], [200]);
         $mock = new Mock([
-            new Response(200, [], Stream::factory('{"a": "'))
+            new Response(200, [], Stream::factory('{"a": bcd"'))
```

Comment:
upravil jsem to kvůli "JsonLint" byl updatovaný na v1.8.0 a začlo to tam dělat tuhle chybu v testu